zion/src/test/scala/edu/uci/ics/cloudberry/zion/model/impl/TestQuery.scala

```diff
@@ -51,6 +62,62 @@ trait TestQuery {
        | ]
      """.stripMargin
 
+  val filterWrongValueJSON =
+    s"""
+       |"filter": [
+       |  {
+       |    "field": "geo_tag.stateID",
+       |    "relation": "in",
+       |    "values": [[${stateValue.mkString(",")}]]
+       |  }
+       | ]
+     """.stripMargin
+
+  val filterWrongRelationJSON =
+    s"""
+       |"filter": [
+       |  {
+       |    "field": "geo_tag.stateID",
+       |    "relation": "iin",
+       |    "values": [${stateValue.mkString(",")}]
+       |  }
+       | ]
+     """.stripMargin
+
+  val filterLongValueJSON =
+    s"""
+       |"filter": [
+       |  {
+       |    "field": "id",
+       |    "relation": "inRange",
+       |    "values": [${longValues.mkString(",")}]
+       |  }
+       | ]
+     """.stripMargin
+
+  val filterDoubleValueJSON =
+    s"""
+       |"filter": [
+       |  {
+       |    "field": "id",
+       |    "relation": "inRange",
+       |    "values": [${doubleValues.mkString(",")}]
+       |  }
+       | ]
+     """.stripMargin
+
+
+  val filterRetweetJSON =
+    s"""
+       |"filter": [
+       |  {
+       |    "field": "is_retweet",
+       |    "relation": "true",
+       |    "values": []
```

Comment:
Currently this is the only case "values" is not required. I can change it to optional for now. But I think a better approach would be later change the boolean relation to "is" and values:[true/false]